src/RecoveryServices/RecoveryServices.Backup.Autorest/custom/Helpers/PolicyHelpers.ps1

```diff
@@ -802,4 +802,80 @@ function ValidateMandatoryFields {
             }
         }
     }
+}
+
+
+
+function ValidateTieringPolicy
+{
+    if($tieringdetails.TieringMode -eq "")
+    {
+        $errormsg="Please specify the tiering mode"
+        throw $errormsg
+    }
+    if(($tieringdetails.TieringMode -eq "DoNotTier") -or ($tieringdetails.TieringMode -eq "TierRecommended"))
+    {
+        if(($tieringdetails.Duration -ne $null -or $tieringdetails.durationType -ne $null) -and ($tieringdetails.Duration -gt 0 -and $tieringdetails.durationType -ne "Invalid"))
+        {
+            $errormsg="Invalid values provided according to TieringMode"
+            throw $errormsg
+        }
+    }
+    elseif($tieringdetails.TieringMode -eq "TierAfter")
+    {
+        if(($tieringdetails.Duration -eq $null) -or ($tieringdetails.Duration -eq 0) -or ($tieringdetails.TieringMode -eq $null) -or ($tieringdetails.TieringMode -eq ""))
```

Comment:
should this be 

if(($tieringdetails.Duration -eq $null) -or ($tieringdetails.Duration -eq 0) -or ($tieringdetails.**DurationType** -eq $null) -or ($tieringdetails.**DurationType**-eq ""))